components/tools/OmeroPy/src/omero/plugins/obj.py

```diff
@@ -301,7 +408,8 @@ def _configure(self, parser):
             "--file", help=SUPPRESS)
         parser.add_argument(
             "command", nargs="?",
-            choices=("new", "update"),
+            choices=("new", "update", "null", "map-get",
+                     "map-set", "map-add", "map-edit"),
```

Comment:
Is the intention to add the subcommands `map-add` and `map-edit` before this PR is merged?